addon/components/nypr-account-forms/signup.js

```diff
@@ -22,7 +22,14 @@ export default Component.extend({
 
   init() {
     this._super(...arguments);
-    set(this, 'newUser', this.get('store').createRecord('user'));
+    set(this, 'newUser', this.get('store').createRecord('user',
+      {
```

Comment:
small JS style note: the opening and close curly braces don't usually get their own line.